src/platform_impl/linux/taskbar.rs

```diff
@@ -0,0 +1,66 @@
+mod unity;
+
+pub struct TaskbarIndicator {
+  unity: Option<unity::Manager>,
+  progress: f64,
+  progress_visible: bool,
+  needs_attention: bool,
+}
+
+#[allow(dead_code)]
+impl TaskbarIndicator {
+  pub fn new() -> Result<Self, Box<dyn std::error::Error>> {
```

Comment:
No need to return a result here, it doesn't contain any fallible operation

```suggestion
  pub fn new() -> Self {
```